src/components/Modals/DeleteCourseModal.vue

```diff
@@ -9,26 +9,31 @@
     @modal-closed="closeCurrentModal"
     :rightButtonIsDisabled="false"
   >
+    <!-- Remove course -->
     <div v-if="isTransferCredit" class="text-width">
-      Are you sure you want to remove "{{ reqName }}" for this requirement? This will delete the
-      selected transfer credit.
+      Are you sure you want to remove <b>reqName</b> from the <b>{{ reqDesc }} Requirement?</b> This
```

Comment:
Did we determine whether these bold tags are having any effect?

Reply:
I think they do make it slightly more bold, but not as bold as the designs make it, so I just made the font larger as well.